### PR TITLE
Allow release-channel and run-id inputs.

### PR DESCRIPTION
**Pull request by doxavore**
I'd like to test these out more thoroughly before adding `run-id` to the README. I'll follow up once that's done.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -13,3 +13,13 @@ inputs:
       RWX Access Token.
       See https://www.rwx.com/docs/access-tokens
     required: true
+  release-channel:
+    description: |
+      ABQ Release Channel.
+    required: false
+    default: v1
```
Comment (ayazhafiz) [inline action.yml:20]: It looks like we wanted to provide the release channel param previously but not expose it publicly: https://github.com/rwx-research/setup-abq/pull/11. Will this still work if we don't expose release-channel here?

Comment (doxavore) [reply action.yml:20]: Yeah, it'll still be publicly exposed, but I think we leave it undocumented for now.

Comment (doxavore) [reply action.yml:20]: To expand: when we don't list it as an input here, GitHub warns when we try to use it, and doesn't seem to pass it to the underlying action. I believe we _must_ declare it here to get it to pass through.

Comment (michaelglass) [reply action.yml:20]: oh I thought it passed it through anyways. So be it

File: src/index.ts [hunk 2]
```diff
@@ -21,6 +21,11 @@ function getArch() {
 }
 
 function getRunId() {
+  const providedRunId = core.getInput('run-id')
```
Comment (michaelglass) [inline src/index.ts:24]: 👍

File: action.yml (hunk 1 above)
Comment (michaelglass) [reply action.yml:20]: going to check this once!

Comment (michaelglass) [reply action.yml:20]: fwiw it looks like it works here https://github.com/rwx-research/captain/pull/1459

I don't have a super strong opinion about keeping this stuff private or not. Just FYI

**Review by michaelglass — APPROVED** [review]
consider removing the params from the `actions.yml` but this is also fine

**Comment by dan-manges** [comment]
@doxavore what's the use case for passing in `run-id` rather than generating it from the environment variables?

**Pull request merged**